telemetry/receivedenevlope.go

```diff
@@ -103,3 +103,37 @@ func (r *SentEnvelope) put(db *sql.DB) error {
 
 	return nil
 }
+
+type ErrorSendingEnvelope struct {
+	CreatedAt    int64        `json:"createdAt"`
+	Error        string       `json:"error"`
+	SentEnvelope SentEnvelope `json:"sentEnvelope"`
+}
+
+func (e *ErrorSendingEnvelope) put(db *sql.DB) error {
+	e.CreatedAt = time.Now().Unix()
```

Comment:
Where is this used?

Reply:
good catch, updated the exec statement below this line to use `e.CreatedAt`